lambda/create_db_users.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_lambda_function" "create_db_users_lambda_function" {
+  count         = local.count_create_db_users
+  function_name = "${var.project}-create-db-users-${local.environment}"
+  handler       = "uk.gov.nationalarchives.db.users.Lambda::process"
+  role          = aws_iam_role.create_db_users_lambda_iam_role.*.arn[0]
+  runtime       = "java8"
```

Comment:
Is it possible to run this on a newer version of Java?

Reply:
Yeah, it can, I copied and pasted this off an old one. I've changed it to java 11.